cabal-install/src/Distribution/Client/ProjectPlanning/Types.hs

```diff
@@ -691,7 +723,7 @@ data ElaboratedPackage
 
        -- | The exact dependencies (on other plan packages)
        --
-       pkgLibDependencies :: ComponentDeps [ConfiguredId],
+       pkgLibDependencies :: ComponentDeps [(ConfiguredId, Bool)],
```

Comment:
Insufficient documentation. At least the boolean value needs to be explained, what its values are supposed to mean.